.github/workflows/testing.yml

```diff
@@ -0,0 +1,22 @@
+name: testing
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - main
+      - 'releases/*'
+
+jobs:
+  test: 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          npm install
+      - run: |
+          npm run build
+      - run: |
+          npm run package
+      - run: |
+          npm run test
+  
```

Comment:
Thie original test.yml also ran the action on `.` why was this deleted?

Reply:
reinstated the `.` without the `wait` since that doesnt exist anymore